packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -181,6 +181,7 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
     const { data, lineLegendColor = theme!.palette.yellow, lineLegendText } = this.props;
```

Comment:
I think you will need same change in VerticalStackedBarChart as well

Reply:
Ok , i will add these changes for the Vertical Stacked Bar chart